src/OSPSuite.Core/Events/AnalysableEvents.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using System.Collections.Generic;
 using OSPSuite.Core.Domain;
 using OSPSuite.Core.Domain.Data;
+using OSPSuite.SimModel;
```

Comment:
Why was this added? if htis is required, this is PROBABLY NOT The right ISimulation here..

Reply:
It got added at one point, but it's not used